WebAPI/Controllers/UserController.cs

```diff
@@ -123,5 +123,55 @@ public IActionResult Renew([FromBody] RenewModel body)
             var authUser = new AuthenticatedUserModel(token);
             return Ok(authUser);
         }
+
+        [AllowAnonymous]
+        [HttpPost("resetPassword")]
+        public async Task<IActionResult> ResetPassword([FromBody] ResetPasswordModel body)
+        {
+            body.Email = body.Email.ToLower();
+            var user = await _userService.GetUserByEmailAsync(body.Email);
+            if (user == null)
+            {
+                _logger.LogInformation($"Invalid password reset attemp. User {body.Email} doesn't exist.");
+                return ErrorResponse($"Používateľ {body.Email} neexistuje.");
+            }
+
+            if (!user.EmailConfirmed)
```

Comment:
What about if I lost my confirmation email and I don't remember my password?

Reply:
User can resend register confirmation email. A button on login page appears when he tries to log in without having email confirmed.